README.md

```diff
@@ -35,8 +35,8 @@ docker run -d --restart=always \
   -e MYSQL_DATABASE=cbioportal \
   -v /<path_to_save_mysql_db>/db_files/:/var/lib/mysql/ \
   -v /<path_to_seed_database>/cgds.sql:/docker-entrypoint-initdb.d/cgds.sql:ro \
-  -v /<path_to_seed_database>/seed-cbioportal_no-pdb_hg19.sql.gz:/docker-entrypoint-initdb.d/seed_part1.sql.gz:ro \
-  -v /<path_to_seed_database>/seed-cbioportal_only-pdb.sql.gz:/docker-entrypoint-initdb.d/seed_part2.sql.gz:ro \
+  -v /<path_to_seed_database>/seed-cbioportal_<genome_name>_<seed_version>.sql.gz:/docker-entrypoint-initdb.d/seed_part1.sql.gz:ro \
```

Comment:
Makes sense: that would be more consistent with the use of the word in other places such as https://github.com/cBioPortal/cbioportal/blob/v1.5.1/docs/Using-the-dataset-validator.md#validation-of-non-human-data